narayana-full/pom.xml

```diff
@@ -186,7 +186,35 @@
       <version>${version.org.jboss.integration}</version>
       <scope>provided</scope>
     </dependency>
-
+    <dependency>
+      <groupId>org.jboss.resteasy</groupId>
+      <artifactId>resteasy-jaxrs</artifactId>
```

Comment:
Can you explain why we needed to add this dependency here please?

Reply:
It needs to generate the restat javadoc. There are warnings with the JDK 8 about can not found the class but the errors with the JDK9+.